SimpleITK/utilities/resize.py

```diff
@@ -28,6 +28,9 @@ def resize(
     isotropic. The physical extent of the image's data is retained in the new image, with the new image's spacing
     adjusted to achieve the desired size. The image is centered in the new image.
 
+    By default the function will perform anti-aliasing. If runtime is of concern (e.g. resizing multiple images
```

Comment:
Change to:

Anti-aliasing is enabled by default.

Performance can be increased by disabling anti-aliasing ( `anti_aliasing_sigma=0` ), and by setting the interpolator to `sitkNearestNeighbor` at the cost of decreasing quality.